sdk/README.md

```diff
@@ -19,8 +19,9 @@ Happy hacking!
 
 ## Aleo Tools
 
-[Aleo SDK account generator](https://aleohq.github.io/aleo/)
+<!-- markdown-link-check-disable -->
```

Comment:
```suggestion
<!-- markdown-link-check-disable -->
```
These link disables should be added on https://github.com/AleoHQ/welcome

Reply:
@collinc97 I intentionally pushed this PR to the SDK repo because the markdown is autogenerated. The link disables will be overwritten every time we regenerate the documentation based on the README.md.